gusto/recovery/recovery_options.py

```diff
@@ -0,0 +1,119 @@
+
+from firedrake import (interval, FiniteElement, TensorProductElement, FunctionSpace,
+                       VectorFunctionSpace)
+from gusto.core.function_spaces import DeRhamComplex
+from gusto.core.configuration import RecoveryOptions
+
+
+class RecoverySpaces(object):
+    """
+    Finds or builds necessary spaces to carry out recovery transport for lowest
+    and mixed order domains (0,0), (0,1) and  (1,0)
+    """
+
+    def __init__(self, domain, boundary_method=None, use_vector_spaces=False):
+        """
+        Args:
+            domain (:class:`Domain`): the model's domain object, containing the
+            mesh and the compatible function spaces.
```

Comment:
Could you indent the continuation lines, so that it's clear which are the variables?